src/ru/yandex/practicum/tasktracker/service/impl/InMemoryHistoryManager.java

```diff
@@ -3,25 +3,117 @@
 import ru.yandex.practicum.tasktracker.model.Task;
 import ru.yandex.practicum.tasktracker.service.HistoryManager;
 
-import java.util.LinkedList;
-import java.util.List;
+import java.util.*;
 
 public class InMemoryHistoryManager implements HistoryManager {
-    private static final int SIZE_OF_REQUEST_HISTORY = 10;
-    private final List<Task> history = new LinkedList<>();
+    private final Map<Integer, Node> requestHistory = new HashMap();
```

Comment:
```suggestion
    private final Map<Integer, Node> requestHistory = new HashMap<>();
```
Лучше параметризовать `HashMap`. Вот [тут](https://www.baeldung.com/raw-types-java) больше про то, почему плохо использовать `raw types`